index.html

```diff
@@ -48,90 +48,8 @@ <h1 class="main-heading">I'm Misal.<br> Glad to see you!</h1>
-            <label class="frame-text2">Back End Dev</label>
-            <div class="frame-counter"><img src="images/Counter.png"></div>
-            <label class="frame-text2">2015</label>
-          </div>
-          <p class="works-paragraph">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
-          <ul class="works-list">
-            <li><label class="list-buttons">HTML</label></li>
-            <li><label class="list-buttons">CSS</label></li>
-            <li><label class="list-buttons">JAVASCRIPT</label></li>
-          </ul>
-          <button class="works-button">See Project</button>
-        </div>
-      </div>
-      <div id="works-1">
-        <div class="img-container"><img src="images/Snapshoot3.png"></div>
-        <div class="img-container-desktop"><img class="desktop-image" src="images/Snapshoot-Desktop1.png"></div>
-        <div class="works-contents">
-          <h2 class="works-heading">Tonic</h2>
-          <div class="works-frame">
-            <label class="frame-text1">Canopy</label>
-            <div class="frame-counter"><img src="images/Counter.png"></div>
-            <label class="frame-text2">Back End Dev</label>
-            <div class="frame-counter"><img src="images/Counter.png"></div>
-            <label class="frame-text2">2015</label>
-          </div>
-          <p class="works-paragraph">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
-          <ul class="works-list">
-            <li><label class="list-buttons">HTML</label></li>
-            <li><label class="list-buttons">CSS</label></li>
-            <li><label class="list-buttons">JAVASCRIPT</label></li>
-          </ul>
-          <button class="works-button">See Project</button>
-        </div>
-      </div>
-      <div id="works-1">
-        <div class="img-container"><img src="images/Snapshoot4.png"></div>
-        <div class="img-container-desktop"><img class="desktop-image" src="images/Snapshoot-Desktop1.png"></div>
-        <div class="works-contents">
-          <h2 class="works-heading">Multi-Post<br> Stories</h2>
-          <div class="works-frame">
-            <label class="frame-text1">Canopy</label>
-            <div class="frame-counter"><img src="images/Counter.png"></div>
-            <label class="frame-text2">Back End Dev</label>
-            <div class="frame-counter"><img src="images/Counter.png"></div>
-            <label class="frame-text2">2015</label>
-          </div>
-          <p class="works-paragraph">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
-          <ul class="works-list">
-            <li><label class="list-buttons">HTML</label></li>
-            <li><label class="list-buttons">CSS</label></li>
-            <li><label class="list-buttons">JAVASCRIPT</label></li>
-          </ul>
-          <button class="works-button">See Project</button>
-        </div>
+      
```

Comment:
Please I ensure you stick to the required [Figma Design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0-1&t=JEEYDTyJXTMKqYe9-0). 😃 

_**REQUIRED FIGMA DESIGN**_

- [ ] _**Desktop version.**_

![Screenshot (223)](https://user-images.githubusercontent.com/78104544/231328148-403d2975-5ae7-4572-a348-6237edba5940.png)

- [ ] _**Mobile version.**_

![Screenshot (228)](https://user-images.githubusercontent.com/78104544/231329099-14d40ff7-7bd0-4308-8142-efabff8a226d.png)


_**YOUR DESIGN**_

- [ ] Please I recommend adjusting the popup content to fit the full height of the screen. 😃 
- [ ] Please I recommend adding a blurry background to the popup content. 😃 

- [ ] _**Desktop version.**_

![Screenshot (224)](https://user-images.githubusercontent.com/78104544/231328188-f53332c5-8a6a-4bb3-a103-a0ad2ecb5487.png)

- [ ] _**Mobile version.**_

![Screenshot (229)](https://user-images.githubusercontent.com/78104544/231329324-a8840a2f-d905-4ed5-be6a-f6122c0e0981.png)

Reply:
@braincee

> Hello Team @misalazeem and @deniodev , I am @braincee your Code Reviewer
> 
> Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!
> 
> ![so-close](https://user-images.githubusercontent.com/78104544/225492039-ed22bba9-481a-489f-ae04-74623970af20.gif) [ ![so-close](https://user-images.githubusercontent.com/78104544/225492039-ed22bba9-481a-489f-ae04-74623970af20.gif) ](https://user-images.githubusercontent.com/78104544/225492039-ed22bba9-481a-489f-ae04-74623970af20.gif) [ ](https://user-images.githubusercontent.com/78104544/225492039-ed22bba9-481a-489f-ae04-74623970af20.gif)
> 
> ## HIGHLIGHTS
> * GitHub flow rightfully used ✅
> * Meaningful title and summary for PR ✅
> * Great implementation of pair programming✅
> * No linter errors ✅
> 
> ## REQUIRED CHANGES ♻️
> **_Please check the comments under the review._**
> 
> ## OPTIONAL CHANGES
> **_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._**
> 
> * [ ]  [OPTIONAL] Please I recommend making the commit messages imperative tenses and capitalized. This is a provided [link](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md) for instructions on how to document your repository professionally. 😃
> * [ ]  [OPTIONAL] Please I highly recommend fixing merge conflicts. 😃
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me @braincee in your question so I can receive the notification.
> 
> _Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

I totally understand the height concerns. But if you look at the Figma design for the desktop it is a higher screen height than the width. Usual Screens we work with are landscape with bigger heights. Now the issue is the content like image have certain dimensions. So I am not sure as to how to fit the whole content box inside the viewport. Any recommendations or suggestions are welcome.

Best Regards,
Misal Azeem